java/src/jmri/jmrix/can/CanSystemConnectionMemo.java

```diff
@@ -104,7 +104,12 @@ public boolean provides(Class<?> type) {
             return mgr.isAddressedModePossible();
         }
         if (type.equals(jmri.ConsistManager.class)) { // until a CAN ConsistManager is implemented, use Internal
-            return false;
+            if (_protocol == ConfigurationManager.SPROGCBUS) {
+                // SPROG CBUS DOES have a consist manager but we'll do things correctly and check with the manager
+                return manager.provides(type);
+            } else {
+                return false;
+        }
```

Comment:
No big deal, but this indentation seems to have gone wrong. Fixing it makes it easier to read the code.

Reply:
Thanks. I made a couple fo other changes to fic some CI errors.